__tests__/app.test.js

```diff
@@ -41,14 +49,46 @@ describe("\n/api/categories tests:\n", () => {
         });
     });
   });
+});
 
-  describe("\nerror handling tests:\n", () => {
-    test("sends 404 - bad request when passed incorrect path", () => {
+describe("\n/api/reviews tests:\n", () => {
+  describe("GET:200 and send array of reviews with comment count added", () => {
+    test("get 200 status and an array of objects", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then((response) => {
+          expect(response.body.reviews).toEqual(expect.any(Array));
+        });
+    });
+    test("all reviews objects are sent", () => {
       return request(app)
-        .get("/api/wrongpath")
-        .expect(404)
+        .get("/api/reviews")
+        .expect(200)
         .then((response) => {
-          expect(response.body.msg).toBe("invalid path");
+          expect(response.body.reviews.length).toBe(13);
+        });
+    });
+    test("all properties returned including new comment_count property", () => {
```

Comment:
Add another 200 test that makes sure the reviews are sorted by date in descending order as a default - jest-sorted will be your friend here :)